pkgs/by-name/im/imager/package.nix

```diff
@@ -0,0 +1,98 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gtk2-x11,
+  pkg-config,
+  python3,
+  gfortran,
+  lesstif,
+  cfitsio,
+  getopt,
+  perl,
+  groff,
+  which,
+  darwin,
+  ncurses,
+}:
+
+let
+  python3Env = python3.withPackages (
+    ps: with ps; [
+      numpy
+      setuptools
+    ]
+  );
+in
+
+stdenv.mkDerivation rec {
+  version = "4.4-01";
+  pname = "imager";
+
+  src = fetchurl {
+    url = "https://dave.oasu.u-bordeaux.fr:5000/5964614a-44f2-47fd-ba18-5f155543044e/imager-last.tgz";
```

Comment:
This... does not look like a stable download link. Are you sure?

Reply:
The recommended download link is on [Nextcloud instance](https://sdrive.cnrs.fr/s/BmXHoY3EgbY6Yre), which requires to accept some general terms of use. I could not find a way to download the source from this link with `curl` (see [this discussion ](https://discourse.nixos.org/t/how-download-file-from-an-url-with-special-characters-in-it/57717)). As a workaround, I used the link [from the MacPort package](https://github.com/macports/macports-ports/blob/master/science/imager/Portfile). I don't know if the link is stable; but the file name `imager-latest` suggest it is not.

Reply:
If it's not stable then I'm afraid the reproducibility guarantees Nixpkgs tries to preserve would be compromised greatly...

Reply:
I'll contact the upstream developers to ask them if they could provide a stable link. Meanwhile, I could host  the source code at my university. Would that be OK?

Reply:
I have setup a [mirror for the source code](https://cloud.univ-grenoble-alpes.fr/s/J6yEqA6yZ8tX9da).